datajoint/expression.py

```diff
@@ -342,6 +346,26 @@ def fetch1(self):
     def fetch(self):
         return Fetch(self)
 
+    def head(self, limit=25, **fetch_kwargs):
+        """
+        shortcut to fetch the first few entries from query expression.
+        Equivalent to fetch(order_by="KEY", limit=25)
+        :param limit:  number of entries
+        :param fetch_kwargs: kwargs for fetch
+        :return: query result
+        """
+        return self.fetch(order_by="KEY", limit=limit, **fetch_kwargs)
+
+    def tail(self, limit=25, **fetch_kwargs):
+        """
+        shortcut to fetch the last few entries from query expression.
+        Equivalent to fetch(order_by="KEY DESC", limit=25)
```

Comment:
not quite - it's equivalent to `fetch(order_by="KEY DESC", limit=25)[::-1]`

Reply:
right